src/database/Database.cpp

```diff
@@ -119,6 +119,7 @@ Database::applySchemaUpgrade(unsigned long vers)
 
     case 4:
         BanManager::dropAll(*this);
+        DataFrame::dropAll(*this); // not and error, it changed again
```

Comment:
this change is actually wrong: deleting from the database doesn't delete from the bucket list so I think this will cause entries to show up as "create" instead of "updates" and potentially cause weird problems down the line with shadow entries. The proper way to do this is to do a alter table where lastmodified is something consistent with the past (I think it was stored as 0 in the xdr with the previous bug)